### PR TITLE
Add Django test environment for interacting with Django's built-in database and templating

### PR DESCRIPTION
**Pull request by snowminc**
#31

### CODE REVIEW

File: app/groupformer/groupformer/urls.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+"""groupformer URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/3.1/topics/http/urls/
```
Comment (High-Tek-Toaster) [inline app/groupformer/groupformer/urls.py:4]: Good comment

**Review by Brunswick-Khorasan — APPROVED** [review]
Demo in meeting worked very well, things look good

**Review by High-Tek-Toaster — APPROVED** [review]
Good stuff yo

**Comment by snowminc** [comment]
Tests written found in:
app/groupformer/project_input_test/tests.py

**Pull request merged**